File: Python/List-2/has22.py
# Given an array of ints, return True if the array contains a 2 next to
# a 2 somewhere.

def has22(nums):
  # Go over the array, looking for a 2 followed by a 2
  for i in range(len(nums) - 1):
    if nums[i] == 2 and nums[i + 1] == 2:
      return True
  
  # There is no 2 followed by another 2, return false
  return False

File: Java/Array-2/has22.java
/*
 * Given an array of ints, return true if the array
 * contains a 2 next to a 2 somewhere.
 */

public boolean has22(int[] nums) {
  // Go over the array, checking if there is a 2 next to a 2
  for(int i = 0; i < nums.length - 1; i++){
    if(nums[i] == 2 && nums[i + 1] == 2)
      return true;
  }
  
  // No 2 followed by a 2 was found, return false
  return false;
}
